components/SearchResultsList.vue

```diff
@@ -10,24 +10,35 @@
                 </svg>
-                <span class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
-                    $t('searchResultsList.filters') }}</span>
+                <span class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">
+                    {{ $t('searchResultsList.filters') }}</span>
             </button>
         </div>
-        <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
-            <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
-                v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
-                <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
-                    hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
-                    <SearchResultsListItem
-                        :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
-                        :degrees="searchResult.professional.degrees"
-                        :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
-                        :specialties="searchResult.professional.specialties"
-                        :spoken-languages="searchResult.professional.spokenLanguages" />
+        <div v-if="resultsStore.searchResultsList.length > 0">
+            <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
+                <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
+                    v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
+                    <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
+                        hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
+                        <SearchResultsListItem
+                            :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
+                            :degrees="searchResult.professional.degrees"
+                            :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
+                            :specialties="searchResult.professional.specialties"
+                            :spoken-languages="searchResult.professional.spokenLanguages" />
+                    </div>
                 </div>
             </div>
         </div>
+        <div v-else>
+            <div class="h-full flex flex-col w-[358px]">
+                <img src="../assets/images/No-search-results-graphic.png" class="p-12" />
```

Comment:
@Tch4lla I just noticed that you're using a png file. We'll want to use an SVG wherever possible. You can get the SVG code by switching Figma to "dev mode" and clicking on an image. I think it should be this:

```
<svg xmlns="http://www.w3.org/2000/svg" width="120" height="69" viewBox="0 0 120 69" fill="none">
<path fill-rule="evenodd" clip-rule="evenodd" d="M56.3139 1.7783C69.4727 0.336104 84.2433 -2.31383 94.5448 4.16156C105.344 10.9498 106.969 22.8959 107 33.6388C107.031 44.4109 105.122 56.06 94.7152 63.2473C84.6399 70.2057 69.8266 69.1089 56.3139 68.6443C43.6453 68.2087 29.6018 68.0371 21.1913 60.7194C13.0343 53.6222 14.8527 43.0742 16.0811 33.6388C17.1518 25.4139 20.1202 17.4214 27.5991 11.4988C35.18 5.49543 45.555 2.95747 56.3139 1.7783Z" fill="#FBF5F0"/>
<path d="M93.7013 22.2582C92.8577 22.2582 92.063 22.1263 91.3173 21.8627C90.5716 21.5991 89.8899 21.2262 89.2723 20.7442L88.0068 21.987C87.8411 22.1527 87.6302 22.2356 87.3741 22.2356C87.118 22.2356 86.9071 22.1527 86.7414 21.987C86.5757 21.8213 86.4928 21.6104 86.4928 21.3543C86.4928 21.0982 86.5757 20.8873 86.7414 20.7216L87.9842 19.4787C87.5021 18.8611 87.1293 18.1756 86.8657 17.4224C86.602 16.6691 86.4702 15.8707 86.4702 15.0271C86.4702 13.0084 87.1707 11.2986 88.5717 9.89755C89.9728 8.49653 91.6826 7.79602 93.7013 7.79602H100.932V15.0271C100.932 17.0458 100.232 18.7556 98.8308 20.1566C97.4298 21.5577 95.72 22.2582 93.7013 22.2582ZM93.7013 20.4504C95.2078 20.4504 96.4883 19.9231 97.5428 18.8686C98.5973 17.8141 99.1246 16.5336 99.1246 15.0271V9.60379H93.7013C92.1948 9.60379 90.9143 10.1311 89.8598 11.1856C88.8052 12.2401 88.278 13.5206 88.278 15.0271C88.278 15.6146 88.3684 16.1758 88.5491 16.7106C88.7299 17.2454 88.9785 17.7312 89.2949 18.1681L93.9725 13.4905C94.1382 13.3248 94.3491 13.2419 94.6052 13.2419C94.8613 13.2419 95.0722 13.3248 95.2379 13.4905C95.4187 13.6713 95.5091 13.8859 95.5091 14.1345C95.5091 14.3831 95.4187 14.5977 95.2379 14.7785L90.5603 19.4561C90.9972 19.7725 91.483 20.0173 92.0178 20.1905C92.5526 20.3638 93.1138 20.4504 93.7013 20.4504Z" fill="#E0791C"/>
<path d="M107.82 44.9079C107.04 44.5877 106.355 44.1641 105.765 43.6371C105.175 43.1101 104.686 42.5064 104.297 41.8259L102.655 42.4953C102.439 42.5857 102.212 42.5823 101.975 42.4851C101.738 42.3879 101.575 42.2311 101.484 42.0149C101.394 41.7987 101.397 41.5721 101.494 41.3352C101.592 41.0983 101.748 40.9346 101.965 40.8442L103.586 40.1662C103.375 39.4118 103.29 38.6361 103.332 37.8392C103.374 37.0423 103.555 36.2536 103.876 35.4731C104.642 33.6055 105.939 32.2896 107.767 31.5253C109.595 30.761 111.443 30.7621 113.31 31.5284L120 34.2735L117.255 40.9632C116.489 42.8308 115.192 44.1467 113.364 44.911C111.536 45.6753 109.688 45.6743 107.82 44.9079ZM108.506 43.2355C109.9 43.8074 111.285 43.8057 112.661 43.2304C114.037 42.6552 115.011 41.6707 115.583 40.277L117.641 35.2597L112.624 33.2008C111.23 32.629 109.846 32.6306 108.47 33.2059C107.094 33.7812 106.12 34.7657 105.548 36.1594C105.325 36.7029 105.196 37.2564 105.16 37.8198C105.124 38.3832 105.169 38.927 105.296 39.4513L111.399 36.8995C111.616 36.8091 111.842 36.8125 112.079 36.9098C112.316 37.007 112.48 37.1637 112.57 37.3799C112.669 37.6158 112.671 37.8487 112.577 38.0787C112.482 38.3086 112.317 38.4729 112.081 38.5715L105.978 41.1233C106.262 41.5818 106.619 41.9927 107.048 42.356C107.477 42.7193 107.963 43.0125 108.506 43.2355Z" fill="#E0791C"/>
<path d="M15.0399 35.1875C15.1138 36.0279 15.0521 36.831 14.8547 37.597C14.6574 38.3629 14.3457 39.0746 13.9196 39.7321L15.2685 40.8838C15.4481 41.0344 15.5491 41.2372 15.5715 41.4923C15.5939 41.7475 15.5299 41.9648 15.3793 42.1444C15.2287 42.324 15.0259 42.425 14.7708 42.4474C14.5157 42.4699 14.2983 42.4058 14.1187 42.2552L12.7718 41.126C12.1988 41.6603 11.5486 42.0918 10.8213 42.4204C10.0941 42.7489 9.31027 42.9502 8.46989 43.0241C6.45897 43.2009 4.69434 42.6528 3.17601 41.3798C1.65767 40.1069 0.810107 38.465 0.633314 36.4541L2.65695e-05 29.2508L7.20331 28.6175C9.21423 28.4407 10.9789 28.9888 12.4972 30.2617C14.0155 31.5346 14.8631 33.1766 15.0399 35.1875ZM13.2391 35.3458C13.1071 33.8451 12.4697 32.6157 11.3269 31.6576C10.1841 30.6995 8.86232 30.2864 7.36164 30.4183L1.95917 30.8933L2.43414 36.2957C2.56607 37.7964 3.20345 39.0258 4.34629 39.984C5.48912 40.9421 6.81088 41.3552 8.31157 41.2232C8.89683 41.1718 9.44792 41.0326 9.96483 40.8057C10.4817 40.5788 10.9439 40.2886 11.3514 39.9352L6.28215 35.6852C6.10257 35.5347 6.00156 35.3318 5.97913 35.0767C5.9567 34.8216 6.02077 34.6042 6.17133 34.4246C6.33558 34.2287 6.54151 34.1199 6.78912 34.0981C7.03674 34.0764 7.2585 34.1476 7.45441 34.3118L12.5237 38.5618C12.8006 38.0989 13.0019 37.5935 13.1276 37.0456C13.2534 36.4977 13.2905 35.9311 13.2391 35.3458Z" fill="#E0791C"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M82.3001 32.3985C82.315 32.461 82.3411 32.547 82.3858 32.659C82.5759 33.1353 82.9955 33.8183 83.6708 34.6656C84.6604 35.9071 86.1124 37.3942 87.9224 38.9407C88.3258 37.7792 88.6181 36.7124 88.7808 35.7929C88.9034 35.1 88.9469 34.5198 88.9206 34.0618C88.8937 33.5928 88.7979 33.3288 88.7131 33.1986C88.2014 32.4131 87.0869 31.8247 85.7131 31.6731C84.4402 31.5328 83.1701 31.7959 82.3001 32.3985ZM89.5657 40.2816C90.1196 38.789 90.5317 37.3763 90.7502 36.1413C90.8895 35.354 90.9554 34.6097 90.9174 33.9471C90.8799 33.2954 90.7376 32.6423 90.3889 32.107C89.4349 30.6424 87.6532 29.875 85.9323 29.6852C84.1994 29.4941 82.2649 29.8555 80.8742 30.9681C80.448 31.309 80.2923 31.7877 80.2812 32.2201C80.2708 32.6258 80.3815 33.0328 80.5283 33.4005C80.8244 34.1422 81.3823 35.0031 82.1069 35.9122C83.2909 37.3977 85.0289 39.146 87.166 40.9181C85.6249 44.6283 83.2544 48.7792 80.7215 51.9092C73.3525 61.0155 61.6705 63.8034 50.037 61.9581C38.3959 60.1115 26.9969 53.6446 20.3928 44.5225C20.069 44.0751 19.4438 43.975 18.9964 44.2989C18.5491 44.6228 18.449 45.248 18.7728 45.6953C25.727 55.3011 37.6263 62.0144 49.7237 63.9334C61.8288 65.8535 74.3257 62.9921 82.2762 53.1673C84.8247 50.0181 87.1861 45.9355 88.7911 42.2098C94.4973 46.5591 102.575 50.8441 110.823 51.2772C111.374 51.3062 111.845 50.8826 111.874 50.331C111.903 49.7795 111.479 49.3089 110.928 49.28C103.222 48.8753 95.5053 44.8233 89.9554 40.5825C89.8242 40.4822 89.6942 40.3819 89.5657 40.2816Z" fill="#E0791C"/>
<path d="M55.8512 36.7203C55.8512 32.907 56.1925 30.1648 56.8751 28.4935C57.5577 26.8223 59.0053 24.998 61.218 23.0208C63.1481 21.326 64.6193 19.8548 65.6315 18.6073C66.6436 17.3598 67.1497 15.9357 67.1497 14.335C67.1497 12.4049 66.5024 10.8042 65.2078 9.53315C63.9131 8.26206 62.1124 7.62652 59.8057 7.62652C57.4047 7.62652 55.5805 8.35622 54.3329 9.81562C53.0854 11.275 52.2027 12.7579 51.6848 14.2644L44.4114 11.1573C45.4 8.14437 47.2125 5.53158 49.8488 3.31895C52.4851 1.10632 55.8041 0 59.8057 0C64.7488 0 68.5503 1.37701 71.2101 4.13103C73.87 6.88505 75.1999 10.1922 75.1999 14.0526C75.1999 16.4064 74.6938 18.419 73.6817 20.0902C72.6695 21.7615 71.0807 23.6563 68.9151 25.7748C66.6083 27.9874 65.2078 29.6705 64.7135 30.8239C64.2191 31.9773 63.972 33.9427 63.972 36.7203H55.8512ZM59.8057 53.6681C58.2521 53.6681 56.9222 53.115 55.8159 52.0086C54.7095 50.9023 54.1564 49.5724 54.1564 48.0188C54.1564 46.4653 54.7095 45.1354 55.8159 44.029C56.9222 42.9227 58.2521 42.3696 59.8057 42.3696C61.3592 42.3696 62.6891 42.9227 63.7955 44.029C64.9018 45.1354 65.4549 46.4653 65.4549 48.0188C65.4549 49.5724 64.9018 50.9023 63.7955 52.0086C62.6891 53.115 61.3592 53.6681 59.8057 53.6681Z" fill="#E0791C"/>
</svg>
```

You'll want to add the SVG the same way that we do for `heartPlusIcon` in `about.vue`. Please note how it's imported in the script tag and used in the template. Ping me on Slack if you have any questions!

Reply:
Thanks for the tip. This has been added now